test/CMakeLists.txt

```diff
@@ -33,6 +33,11 @@ if (NOT EXCLUDE_JOBS)
     list(APPEND DC_SRC ${JOBS_SRC})
 endif ()
 
+if (NOT EXCLUDE_ST)
+    file(GLOB ST_SRC "../source/tunneling/*.cpp")
+    list(APPEND ST_SRC ${ST_SRC})
+endif ()
```

Comment:
Why is this needed in addition to the statements at line 41?

Reply:
Thanks, not sure how that got duplicated.